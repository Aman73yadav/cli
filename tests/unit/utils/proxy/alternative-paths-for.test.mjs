import { describe, expect, test } from 'vitest'

import alternativePathsFor from '../../../../src/utils/proxy/alternative-paths-for.mjs'

describe('alternativePathsFor', () => {
  test('folder url', () => {
    expect(alternativePathsFor('/folder/')).toMatchSnapshot()
  })

  test('file url', () => {
    expect(alternativePathsFor('/file')).toMatchSnapshot()
  })

  test('file in subfolder url', () => {
    expect(alternativePathsFor('/folder/index.html')).toMatchSnapshot()
  })
})
