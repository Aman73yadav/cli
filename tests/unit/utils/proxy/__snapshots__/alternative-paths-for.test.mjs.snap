// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`alternativePathsFor > file in subfolder url 1`] = `[]`;

exports[`alternativePathsFor > file url 1`] = `
[
  "/file.html",
  "/file.htm",
  "/file/index.html",
  "/file/index.htm",
]
`;

exports[`alternativePathsFor > folder url 1`] = `
[
  "/folder.html",
  "/folder.htm",
  "/folder/index.html",
  "/folder/index.htm",
]
`;
